schedule/virt_autotest/slem-kubevirt-tests.yaml

```diff
@@ -0,0 +1,47 @@
+name:         slem-kubevirt-tests
+description:  >
+    Maintainer: Nan Zhang <nan.zhang@suse.com> qe-virt@suse.de
+    Run kubevirt tests on SLE Micro
+vars:
+    VIDEOMODE: text
+    SYSTEM_ROLE: kvm
+    OPT_KERNEL_PARAMS: hugepagesz=2M hugepages=64 max_loop=256
```

Comment:
Have you verified these vars put here ? Do they really take effect ?

Reply:
Yes, they are verified and they were moved out from yaml file in the latest comment.